src/main/resources/application.yml

```diff
@@ -47,6 +48,8 @@ spring:
    rest:
       # this points at the local WireMock server in the test environment.
       iexBasePath: http://localhost:${wiremock.server.port}/
+      iexHistoricalPath: https://cloud.iexapis.com/stable
```

Comment:
It looks like your test environment points at the real IEX API right now. If you turned off your internet and tried to run your test, it would probably fail. It should point at the local WireMock server